pyfar/dsp/dsp.py

```diff
@@ -355,3 +355,66 @@ def _cross_fade(first, second, indices):
     result = first * window_first + second * window_second
 
     return result
+
+
+def pad_zeros(signal, pad_width, mode='after'):
+    """Pad a signal with zeros in the time domain.
+    The function supports multiple padding modes, either after the last sample,
+    before the first sample, or in the middle. Note that the mode 'before'
+    results in a linear phase shift corresponding to the number of added
+    samples. The mode 'center' can be used to pad signals with a symmetry with
+    respect to the time ``t=0``.
```

Comment:
Since this is also mentioned below, I would keep this shorter or maybe leave it out completely.

Reply:
agree